docuploader/__main__.py

```diff
@@ -111,6 +112,10 @@ def upload(
     if not metadata.language:
         raise Exception("Metadata field 'language' is required.")
 
+    # make final updates to java docs before upload
+    if metadata.language.lower() == "java":
+        prepare_java_toc(documentation_path + "/toc.yml", metadata.name.lower())
```

Comment:
is it Java specific to have the uid names to be all lowercase? Seeing other languages slightly concerns me with using `metadata.name.lower()`

Reply:
this is probably overkill here and I was just including because I did with the language. I doubt it won't come through lowercase. Package names in Java should be all lowercase though

Reply:
I do think the language.lower is needed because that gets generated with createmetadata and someone could put "Java" and then it wouldn't get caught here. I can take out the name.lower because its overkill

Reply:
`language.lower` is fine and needed indeed!